packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -38,26 +38,33 @@ const apiConfig = {
  */
 const getLocale = (locals = {}) => {
     let {originalUrl} = locals
-    const {locale: localeType} = getUrlConfig()
 
+    const {
+        app: {
+            url: {locale: localePosition, site: sitePosition}
+        }
+    } = getConfig()
```

Comment:
I'm assuming you'll clean this up once you are done merging in your addition of the `jsonpath-plus` library.